openwrt-openwisp-monitoring/files/monitoring.init

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh /etc/rc.common
+
+START=99
+STOP=15
+USE_PROCD=1
+PROG="/usr/sbin/openwisp_monitoring"
+PROG_NAME="OpenWISP monitoring daemon"
+
+time_to_seconds() {
+    time=$1
+
+    { [ "$time" -ge 1 ] 2>/dev/null && seconds="$time"; } ||
+        { [ "${time%s}" -ge 1 ] 2>/dev/null && seconds="${time%s}"; } ||
+        { [ "${time%m}" -ge 1 ] 2>/dev/null && seconds=$((${time%m} * 60)); } ||
+        { [ "${time%h}" -ge 1 ] 2>/dev/null && seconds=$((${time%h} * 3600)); } ||
+        { [ "${time%d}" -ge 1 ] 2>/dev/null && seconds=$((${time%d} * 86400)); }
+
+    echo $seconds
+    unset seconds
+    unset time
+}
+
+config_openwisp() {
+    config_get base_url "$1" url
+    config_get uuid "$1" uuid
+    config_get key "$1" key
+    config_get_bool verify_ssl "$1" verify_ssl "0"
```

Comment:
For security should we opt to verify SSL?

Reply:
yes, thanks for spotting!